app/views/parent_objects/_form.html.erb

```diff
@@ -16,54 +16,9 @@
     <%= form.text_field :oid %>
   </div>
 
-  <div class="field">
```

Comment:
Does this means that there is now no view (show or edit) of these fields?. That seems like step backwards to me.  Maybe leave this partial alone stop having the new.html.erb reference it (so new and edit are independent for now)?

Reply:
I think maybe we need more info on requirements, etc., so I'm reverting this change so we can have more discussion about what we want these to look like.